Fix:

error: assigning to 'char *' from incompatible type 'const char *'

This fix is included in 1.2.6.2.

There is no commit URL because the developer did not start using git
until after making this fix.
--- wxui/lisaem_wx.cpp.orig	2007-12-05 18:02:58.000000000 -0600
+++ wxui/lisaem_wx.cpp	2020-10-16 19:01:31.000000000 -0500
@@ -1745,9 +1745,10 @@
     // override config file on startup if passed a parameter.
     if (argc>1)
     {
-      char *ext;
       argv1.Printf(_T("%s"),argv[1]);
-      ext=strstr(argv1.fn_str(),".lisaem");
+      const char *ext=strstr(
+                  (const char *)(argv1.fn_str()),
+                  (const char *)(".lisaem"));
 
       if (ext!=NULL && strlen(ext)==7)  // ensure the proper extension is contained.
          {
